### DISCUSSION TITLE
上一整周都登不上蓝灯，申请退款！

### DISCUSSION THREAD

**Discussion by Red-Lin23** [getlantern/lantern-forum-cn#847] [General]
五月份的时候购买的蓝灯，刚开始就经常用着用着掉线了，还能上勉强凑合着用。上周一整周每天都试登不上，严重影响使用！！麻烦申请退款。账号：543253548@qq.com  @Mike777Khoo

**Comment by MaxMKMax** [comment]
@Red-Lin23 您好. 请问您可以提供我支付单据吗? 谢谢. 
然后帮我们提交个log应用内报告. 我们也想找原因.

**Comment by Red-Lin23** [comment]
你好，这个是我的单据。log报告我不是很清楚在哪里找到，你可以给我一个找到这个文件的
方法？

---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年9月9日(周一) 晚上11:35
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 上一整周都登不上蓝灯，申请退款！ (Discussion #847)




 
@Red-Lin23 您好. 请问您可以提供我支付单据吗? 谢谢.
 然后帮我们提交个log应用内报告. 我们也想找原因.

**Reply by Red-Lin23** [reply]
@Mike777Khoo

**Comment by Red-Lin23** [comment]
你好这是我的单据，log文件麻烦告知一下获取的路径，谢谢～以及后续会如何处理



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年9月9日(周一) 晚上11:35
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 上一整周都登不上蓝灯，申请退款！ (Discussion #847)




 
@Red-Lin23 您好. 请问您可以提供我支付单据吗? 谢谢.
 然后帮我们提交个log应用内报告. 我们也想找原因.

**Reply by Red-Lin23** [reply]
@Mike777Khoo 邮件里单据已经发给你了，目前还没收到你们后续的处理，麻烦回复一下

**Comment by Red-Lin23** [comment]
@Mike777Khoo 一周过去了退款进度怎么样了？
